src/main/cpp/RobotContainer.cpp

```diff
@@ -4,18 +4,131 @@
 
 #include "RobotContainer.h"
 
-RobotContainer::RobotContainer() : m_autonomousCommand(&m_subsystem) {
-  // Initialize all of your commands and subsystems here
+#include <frc/smartdashboard/SmartDashboard.h>
 
-  // Configure the button bindings
-  ConfigureButtonBindings();
+RobotContainer::RobotContainer()
+{
+    // Initialize the hardware.
+    VOKC_CALL(this->InitHardware());
+
+    // Initialize subsystems.
+    VOKC_CALL(this->InitDrivetrain());
```

Comment:
They perform integrity checks (CHECK) and call functions (CALL) with bool return types. If we get an error, we can return false (using OKC_CHECK or regular return) and then OKC_CALL will also return false with an error and so on. This serves as a really easy way for us to do our null checks because then we can see where we have problems